R/Orders.R

```diff
@@ -78,37 +78,44 @@ orders <-
     if (.is_id) {
       # if it's an order_id
       .o_id <- symbol_id

```

Comment:
I think we should move `.o_id <- symbol_id` above the conditional and then use `.o_id`  for all subsequent interaction that is currently changing `symbol_id`. 
I generally try not to overwrite/change the user supplied values unless the function is really simple because it makes debugging more difficult in a large function when you open a debugging browser and don't know what the original user input was because it's been overwritten upstream in the code.